user/cron-jobs.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cron Jobs
+layout: en
+permalink: /user/cron-jobs/
+---
+
+**Please note that cron jobs are not enabled by default.
+Please ask us to unlock it for your repository:
+[support@travis-ci.com](mailto:support@travis-ci.com?subject=Cron)**
+
+<div id="toc"></div>
+
+## Preparation
+
+Travis CI supports building projects time-controlled every day,
+week or month no matter if there was a push on this repository or not.
+This feature is not enables by default. Please contact us if you need it:
+
```

Comment:
Can we change this to something like 

    Travis CI lets you schedule builds at regular intervals independently of whether any commits were pushed to the repository.